mapie/tests/test_classification.py

```diff
@@ -804,6 +746,21 @@
     random_state=random_state,
 )
 
+LARGE_COVERAGES = {
```

Comment:
Could we make a note here of why we have `COVERAGES` and `LARGE_COVERAGES`?

Reply:
Done 👍